.github/workflows/test.yaml

```diff
@@ -0,0 +1,20 @@
+name: 'Stale issues and PR'
+on:
+  workflow_dispatch:
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: harupy/stale@mlflow-stale-bot
+        with:
+          stale-issue-message: 'This issue is stale because it has been open 1 days with no activity. Remove stale label or comment or this will be closed in 1 days.'
+          close-issue-message: 'This issue was closed because it has been stalled for 1 days with no activity.'
+          days-before-stale: 1
+          days-before-close: 1
+          start-date: '2022-06-16T00:00:00Z'
+          days-before-pr-stale: -1
+          days-before-pr-close: -1
+          # Set to 0 for testing
+          days-since-issue-created: 0
+          days-since-last-comment-created: 0
```

Comment:
```suggestion
          minutes-since-last-comment-created: 180
```

Reply:
We can do this, but I'm not sure if it's useful.

Reply:
it helps to speedup demo test.

Reply:
I see. I think we can set `days-since-last-comment-created` to 0 when we want to test.